appinfo/info.xml

```diff
@@ -3,7 +3,12 @@
       xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
     <id>sharepoint</id>
     <name>SharePoint Backend</name>
-    <summary>SharePoint Backend for External storages</summary>
+    <summary>SharePoint Backend for External storages
+      The Sharepoint Backend allows administrators to add Sharepoint document libraries as folders in Nextcloud. This offers an easy way for users to access Sharepoint data in the same place where they find their other files, facilitating collaboration and sharing within and across the borders of the organization. Users can use the desktop client, mobile apps or web interface and comment, tag, share and collaboratively edit files on Sharepoint just like with any other data on Nextcloud.
+          
+Requires Nextcloud 10 or newer running on a Linux Server. Supports Sharepoint 2013 and 2016. Nextcloud accesses Sharepoint through the Sharepoint REST API and uses NTLM authentication. Nextcloud respects file access permissions associated with its configured user credentials. Versioning and sharing are handled by Nextcloud.
```

Comment:
> or newer running on a Linux Server

says it could run anywhere else, which it doesn't. So, let's keep it out? OK otherwise.